components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/CertificatesApi.java

```diff
@@ -0,0 +1,149 @@
+package org.wso2.carbon.apimgt.rest.api.publisher;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.*;
+import org.wso2.carbon.apimgt.rest.api.publisher.CertificatesApiService;
+import org.wso2.carbon.apimgt.rest.api.publisher.factories.CertificatesApiServiceFactory;
+
+import io.swagger.annotations.ApiParam;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.CertificateInfoDTO;
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.CertMetadataDTO;
+import java.io.File;
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.CertificatesDTO;
+
+import java.util.List;
+
+import java.io.InputStream;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.*;
```

Comment:
Here as well, Import the specific package.

Reply:
Please ignore the comment since the codes are generated by swagger maven plugin.